src/test/java/api/CirculationRulesAPITests.java

```diff
@@ -422,7 +421,8 @@ public void canReportValidationError() {
 
     JsonObject json = new JsonObject(response.getBody());
 
-    assertThat(json.getString("message"), containsStringIgnoringCase("tab"));
+    assertThat(json.getString("message"),
+      is("Tabulator character is not allowed, use spaces instead."));
```

Comment:
I don't remember this in the original PR, though it seems to be benign.

Reply:
Seems with the new version of org.folio:testing there is no `containsStringIgnoringCase` method.

Reply:
Got it